src/common/freeze/deepfreeze.ts

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
From muy side not 100% sure what's going on with dates and why they cannot be immutabe